stac-fastapi.tf

```diff
@@ -0,0 +1,12 @@
+module "stac-fastapi" {
+  count = var.deploy_stacfastapi == true ? 1 : 0
+  source = "./modules/stac-fastapi"
+
+  namespace_annotations = {
+    "linkerd.io/inject" = "enabled"
```

Comment:
This should be `var.namespace_annotations` otherwise you're never using that variable.

Reply:
Whoops - this is not the stac fastapi module main file so my comment is totally wrong. Ignore me!